tests/assets/131-ns-deleted-when-bundle-deleted.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: fleet.cattle.io/v1alpha1
+kind: GitRepo
+metadata:
+  name: test-ns-deleted-when-bundle-deleted
+  annotations:
+    {}
+  labels:
+    {}
+  namespace: fleet-local
+spec:
+  branch: main
+  correctDrift:
+    enabled: false
+  insecureSkipTLSVerify: false
+  paths:
+    - bundles-delete-namespaces
+  repo: https://github.com/fleetqa/fleet-qa-examples-public/
+  targets:
+    - clusterSelector:
+        matchExpressions:
+          - key: provider.cattle.io
+            operator: NotIn
+            values:
+              - harvester
```

Comment:
This too as well, as `GitRepo` by default targeting to all clusters under namespace using above `clusterSelector` block